torch/tensor.py

```diff
@@ -414,6 +414,18 @@ def __rfloordiv__(self, other):
     __ge__ = _C._TensorBase.ge
     __abs__ = _C._TensorBase.abs
 
+    def __std_mean__(self, dim=None, unbiased=True, keepdim=False):
```

Comment:
std_mean/var_mean is declared as function only, so to avoid failures during testing on Tensor I had to implement 'fake' functions. It's similar to '__rpow__' function.

Reply:
but that's very different -- `__rpow__` is a real python function, see https://docs.python.org/2.0/ref/numeric-types.html, not a made up function for testing.

I think the issue here is actually that common_methods_invocations.py doesn't work on methods.  We should fix that!  Can you take a look?

For the purposes of this PR, I'm okay with making a fake method, i.e. `_std_mean` that just calls your function `std_mean` and you can write a comment about the situation.